NitroxPatcher/Patches/Dynamic/FreecamController_FreecamToggle_Patch.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    public class FreecamController_FreecamToggle_Patch : NitroxPatch, IDynamicPatch
+    {
+        private static readonly MethodInfo TARGET_METHOD = Reflect.Method((FreecamController t) => t.FreecamToggle());
+
+        public static void Postfix(FreecamController __instance)
+        {
+            Resolve<LocalPlayer>().FreecamEnabled = __instance.mode;
```

Comment:
What's the point of the state syncing of freecam ? Seems totally out of scope to me

Reply:
![image](https://user-images.githubusercontent.com/10561268/151888109-cf6703c5-ee46-4307-93d9-e1dc47e486c5.png)

This method is called by freecam command, this should help remove useless state patch

Reply:
i need to verify but doesn't sitting on the bench also sets you as locked ?